coda/coda_mdstore/presentation.py

```diff
@@ -76,11 +80,59 @@ def getFileHandle(codaId, codaPath):
         except Exception as e:
             exceptionList.append(str(e))
             pass
-    raise Exception(
+    raise FileHandleError(
         "Unable to get handle for id %s at path %s" % (codaId, codaPath)
     )
 
 
+def generateBagFiles(identifier, proxyRoot, proxyMode):
+    """
+    Return list of files in the bag
+    """
+    pathList = []
+    transList = []
+    try:
+        handle = getFileHandle(identifier, "manifest-md5.txt")
+    except FileHandleError:
+        raise FileHandleError("Unable to get handle for id %s" % (identifier))
+    bag_root = handle.url.rsplit('/', 1)[0]
+    line = handle.readline()
+    # iterate over handle and append urls to pathlist
+    while line:
+        line = line.strip()
+        parts = line.split(None, 1)
+        if len(parts) == 2:
+            pathList.append(parts[1])
+        line = handle.readline()
+    # iterate top files and append to pathlist
+    try:
+        topFileHandle = getFileHandle(identifier, "")
+        topFiles = getFileList(topFileHandle.url)
+        for topFile in topFiles:
+            pathList.append(topFile)
+    except:
```

Comment:
Is this a place where we would want to be catching `FileHandleError` specifically, or is it some other exception we might be expecting?